src/main/java/course_project/services/SeaBattle.java

```diff
@@ -0,0 +1,26 @@
+package course_project.services;
+
+import course_project.services.conversation.DialogueMenu;
+
+/**
+ *  Start of SeaBattle game;
+ */
+public final class SeaBattle {
+
+    private SeaBattle() {
+    }
+
+    private static class SeaBattleHelper {
+        private static final SeaBattle INSTANCE = new SeaBattle();
+    }
+
+    public static SeaBattle getInstance() {
+        return SeaBattleHelper.INSTANCE;
+    }
+
+    public void run() {
+        DialogueMenu.printGreetings();
+        DialogueMenu.printMainMenu();
```

Comment:
The method name doesn't correlate with what it does

Reply:
Fully agree, but i couldn't find better for this.
Before the game process started it's fully controlled by the menu - so User picks whatever wants,  that's why it's like an entrance point into the game in this moment.